using-valtimo/tasks/task-list-columns.md

```diff
@@ -0,0 +1,51 @@
+# Task list columns
+Task list columns can be configured to change the visible columns in the task list.
+
+## Configuration
+
+### Autodeployment
+Task list columns can be autodeployed by adding json files on the classpath.
+These files should end with `.case-task-list.json` to be eligible for autodeployment.
+
+Every deployment file for represents a changeset. These files are required to contain a changesetId that should be unique
```

Comment:
> Every deployment file for represents a changeset.

Feels like there is a word missing there.

Reply:
We might want backticks around `changesetId ` as it is an identifier.

Reply:
Please check again.